fusionfrontend/src/app/components/content/locations-map/locations-map.component.html

```diff
@@ -40,8 +40,8 @@
         <agm-marker
             *ngFor="let location of locations; let i = index"
             [agmFitBounds]="true"
-            [latitude]="location.latitude"
```

Comment:
In Zeile 29-30 könnte man das ? vermutlich auch nachziehen

Reply:
Hier wird schon in zeile 26 geprüft ob das object existiert